### PR TITLE
Update dependencies 

### PR DESCRIPTION
**Pull request by tonistayhungrystayfoolish**
## Description

Update dependencies:
-Pigeon
-Flutter sdk
-Carrousel_slider
-RxDart

## Checklist

Before creating any Pull Request, confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`).

- [X ] 📕 I read the [Contributing page](https://github.com/Apparence-io/camera_awesome/blob/master/CONTRIBUTING.md).
- [X] 🤝 I match the actual coding style.
- [X] ✅ I ran ```flutter analyze``` without any issues.

## Breaking Change

- [X ] 🛠 My feature contain breaking change.

Carrousel_slider is not supported and with last version of flutter 3.24 the library crash, because carrousel_slider name have conflict with material.

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -8,11 +8,11 @@ homepage: https://Apparence.io
 repository: https://github.com/Apparence-io/camera_awesome
 
 environment:
-  sdk: ">=3.1.0 <4.0.0"
+  sdk: '>=3.2.6 <4.0.0'
   flutter: ">=1.10.0"
 
 dependencies:
-  carousel_slider: ^4.2.1
```
Comment (kuhnroyal) [inline pubspec.yaml:15]: The `carousel_slider` package got a 5.0.0 release just the other day and is much more widely used than the fork. Probably sticking with the original is the better choice here.

Comment (millerf) [reply pubspec.yaml:15]: @kuhnroyal @biklas7
Done in https://github.com/Apparence-io/CamerAwesome/pull/504 (I branched from your branch @tonistayhungrystayfoolish )

Comment (GTechMicah) [reply pubspec.yaml:15]: @kuhnroyal we now have three open prs on this repo that claim to fix this issue. When are we going to get an update on it being fixed? Your package is now incompatible with the **latest stable** version of flutter.

Comment (kuhnroyal) [reply pubspec.yaml:15]: @GTechMicah I am not involved here. I only looked over the open PR since I also need the change.

Comment (GTechMicah) [reply pubspec.yaml:15]: Oh Sorry, thanks for clarifying

**Pull request merged**